uint8 label # label information is ground or not
bool is_height_valid # the height value is valid or not
float32 height # in height information to the ground plane